test/IntegrationTests/BuildTests.cs

```diff
@@ -46,6 +46,7 @@ public Task DistributionStructure()
 
         return Verifier.Verify(relativesPaths)
             .UseTextForParameters(systemName)
+            .UseDirectory("./snapshots")
```

Comment:
VS after changes:
![image](https://user-images.githubusercontent.com/5972917/198504193-0920a919-6811-46d3-9d31-7a630d8c129b.png)

VS before changes:
![image](https://user-images.githubusercontent.com/5972917/198504267-a85b2461-2fd4-40b2-aaaf-1dd75ee0a68d.png)


I am not sure if I want all files under snapshots folder as all of them are grouped by IDE.

Reply:
Hmm, my first preference is always to keep source dir clean and tidy.  Currently it will become a real mess. 
Wondering if this kind of grouping is configurable, so it will work anyway but based on snapshots dir.

Anyway highly subjective topic, I'll keep the comment open for other opinions.

Reply:
I agree with @Kielek . I definitely prefer the "original" behavior so that the expectations are closer to the test.

Reply:
+1 for @Kielek's suggestion